GVFS/GVFS.FunctionalTests/FileSystemRunners/BashRunner.cs

```diff
@@ -53,6 +53,14 @@ public BashRunner()
             }
         }
 
+        private enum FileType

```

Comment:
I concur that all directories are also files. We're not totally consistent everywhere, but if we want to make a change, I would want to go towards more consistency on directories-as-files.

Reply:
I agree that we should try to be consistent whatever we do.  Since this is private would it make any sense to use `FileAttributes` enum instead.  Doesn't look like there is something specific for symlinks other than ReparsePoint so maybe that doesn't quite fit.  Just throwing it out there.

Reply:
> Since this is private would it make any sense to use FileAttributes enum instead. Doesn't look like there is something specific for symlinks other than ReparsePoint so maybe that doesn't quite fit. Just throwing it out there.

I think in this case we're probably best with our own enum as this is really just to control how we're checking for a few specific types of files.  Perhaps a better name for the enum would be a better option.

Reply:
Yeah this is good for now and if we need something different we can revisit it.